### ISSUE TITLE
Sample dataset contains table columns misclassified as Category 

### ISSUE THREAD

**Issue by attekei** [metabase/metabase#4565]
With a fresh installation of Metabase of the current `master`, many table columns of the sample dataset like `Price` and `Ean` are misclassified as Categories. 

<img width="1100" alt="screen shot 2017-03-21 at 13 04 18" src="https://cloud.githubusercontent.com/assets/6034928/24168293/f8a43366-0e36-11e7-89ce-37cff4902098.png">

This leads to a confusing user experience in query builder as columns behave differently from what you would expect.

**Comment by camsaul** [comment]
I agree that it's weird behavior, but it's intentional. Fields with less than 300 distinct values are automatically marked as categories

**Comment by attekei** [comment]
Got it @camsaul, I will create a separate issue that describes a bug with numerical columns marked as Category.

**Comment by salsakran** [comment]
strictly speaking, this might be a consequence of how we mark columns but I wouldn't call it intentional. 

It's a pretty weird consequence and we should probably tighten up how we deal with "category" types.

**Comment by mazameli** [comment]
This one drives me nuts. It's an annoying UX problem, because it means that when someone is playing around with the sample dataset on a fresh installation, some of our features don't seem to work correctly, where in fact it's because a bunch of numeric fields are misclassified.

**Comment by sbelak** [comment]
Only price is explicitly marked as category. Shall I change it?

**Comment by salsakran** [comment]
Yeah, that sounds like a mis-classification.

**Comment by sbelak** [comment]
K. I'll change it next time I regenerate the dataset.
